flow/peerdbenv/dynamicconf.go

```diff
@@ -107,6 +107,14 @@ var DynamicSettings = [...]*protos.DynamicSetting{
 		ApplyMode:        protos.DynconfApplyMode_APPLY_MODE_NEW_MIRROR,
 		TargetForSetting: protos.DynconfTarget_ALL,
 	},
+	{
+		Name:             "PEERDB_BINARY_FORMAT",
+		Description:      "Binary field encoding; either raw, hex, or base64",
+		DefaultValue:     "base64",
```

Comment:
change to raw as a TODO

Reply:
by time we change default to raw we can probably remove this option